### PR TITLE
Fix for FNF chords showing as split notes in the path notation.

### PR DESCRIPTION
**Pull request by mattfreg**
Currently CHOpt has FNF notes not counting chords properly, this is because it considers chords split notes since the actual game allows you to hit the individual notes in chords separately on non-pro instruments.

This leads to the path notation to be really hard to follow across all instruments. Removing the split note check for Festival fixes this issue.

### CODE REVIEW

**Comment by GenericMadScientist** [comment]
Pulled this and after a semicolon to make this compile, I found it didn't work. Looks like your change makes it not count chords as extra points, see this run of the Jimmy Eat World version of The Middle on lead for example.
path

**Comment by mattfreg** [comment]
Ah, I guess I didn't account for that. My bad! 

I probably won't figure out a fix for this then.
